Client/lib/testAPI.html

```diff
@@ -0,0 +1,52 @@
+<!-- 
+
+
+things to look into 
+loadGeoJson 
+
+
+-->
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <style type='text/css'>
```

Comment:
Yes it can. For testing I was aiming for the smallest footprint so I didn't separate them.